wiki/Glossary/en.md

```diff
@@ -453,7 +440,7 @@ If a jump is too wide, a hyperdash may be used to prevent impossible jumps, thus
 
 ### Katu
 
-Katu, or "Beat", is when you complete a comboset with a 300 (or 100) while getting a 100 or worse throughout that comboset.
+Katu, or beat, is when you complete a comboset with a 300 (or 100) while getting a 100 or worse throughout that comboset.
```

Comment:
"when you complete a particular combo without achieving the highest level of accuracy throughout that set."